src/pages/rechner/[frage].tsx

```diff
@@ -116,6 +123,21 @@ const Frage: NextPage<Props> = ({
   );
 };
 
+const submitAnswers = async (questionIDs: string[]) => {
+  const answers = questionIDs?.map((id) => getLocalStorage(id) || '');
+  const id = uuidv4();
+  const response = await fetch('/api/save-response', {
+    method: 'POST',
+    body: JSON.stringify({ id, answers }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  const data = await response.json();
+  console.log(data);
```

Comment:
Remove `console.log`

also it would be nicer if you could submit only the current answer the user is giving. Doesn't really make sense to always save every question again.